app/src/main/java/edu/nextstep/camp/calculator/MainActivity.kt

```diff
@@ -1,46 +1,94 @@
+
+        initViewBinding()
+        initOperandButtons()
+        initOperatorButtons()
+        initDeleteButton()
+        initEqualsButton()
+    }
+
+    private fun initViewBinding() {
         binding = ActivityMainBinding.inflate(layoutInflater)
         setContentView(binding.root)
+    }
 
-        binding.button0.setOnClickListener {
-            binding.textView.text = "0"
+    private fun initOperandButtons() {
+        (0..9).forEach { number ->
+            val viewId = resources.getIdentifier("$BUTTON_VIEW_PREFIX$number", "id", packageName)
+            val button = findViewById(viewId) as? Button ?: return@forEach
+            setOperandButtonListener(button, Operand(number))
         }
-        binding.button1.setOnClickListener {
-            binding.textView.text = "1"
-        }
-        binding.button2.setOnClickListener {
-            binding.textView.text = "2"
-        }
-        binding.button3.setOnClickListener {
-            binding.textView.text = "3"
-        }
-        binding.button4.setOnClickListener {
-            binding.textView.text = "4"
-        }
-        binding.button5.setOnClickListener {
-            binding.textView.text = "5"
+    }
+
+    private fun setOperandButtonListener(button: Button, operand: Operand) {
+        button.setOnClickListener {
+            binding.textView.text = stringExpressionState.addElement(operand).toString()
         }
-        binding.button6.setOnClickListener {
-            binding.textView.text = "6"
+    }
+
+    private fun initOperatorButtons() {
+        listOf(
+            binding.buttonPlus to Operator.PLUS,
+            binding.buttonMinus to Operator.MINUS,
+            binding.buttonMultiply to Operator.MULTIPLY,
+            binding.buttonDivide to Operator.DIVIDE,
+        ).forEach { (button, operator) ->
+            setOperatorButtonListener(
+                button = button,
+                operator = operator
+            )
         }
-        binding.button7.setOnClickListener {
-            binding.textView.text = "7"
+    }
+
+    private fun setOperatorButtonListener(button: Button, operator: Operator) {
+        button.setOnClickListener {
+            binding.textView.text = stringExpressionState.addElement(operator).toString()
         }
-        binding.button8.setOnClickListener {
-            binding.textView.text = "8"
+    }
+
+    private fun initDeleteButton() {
+        binding.buttonDelete.setOnClickListener {
+            binding.textView.text = stringExpressionState.removeElement().toString()
         }
-        binding.button9.setOnClickListener {
-            binding.textView.text = "9"
+    }
+
+    private fun initEqualsButton() {
+        binding.buttonEquals.setOnClickListener {
+            runCatching {
+                StringCalculator.calculate(stringExpressionState)
+            }
+                .onSuccess {
+                    binding.textView.text = it.value.toString()
+                }
```

Comment:
람다 안에 한 줄씩 들어가면 한 라인으로 적어보는 것은 어떨까요?
```suggestion
            runCatching { StringCalculator.calculate(stringExpressionState) }
                .onSuccess { binding.textView.text = it.value.toString() }
```